### PR TITLE
Auto-set isRelease in release script

### PR DESCRIPTION
**Pull request by ashawley**
Hey Erik,

The release script looks sharp.  The only thing I can think of was making the script handle toggling the `isRelease` value.  There's a benefit to the manual step that you had, but it's possible to automate it.  No hard feelings if you want to keep the manual step.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -25,14 +25,16 @@ lazy val sharedSettings = MimaSettings.settings ++ scalaVersionSettings ++ Seq(
 
   name := "scalacheck",
 
+  isRelease := false,
```
Comment (non) [inline build.sbt]: I'm assuming that the command-line overrides this?

File: release.sh [hunk 2]
```diff
@@ -35,23 +33,15 @@ runsbt() {
     sbt "$1"
     RES=$?
     if [ $RES -ne 0 ]; then
-        echo "sbt '$1' failed: $RES"
+        echo "sbt 'set every isRelease := true' '$1' failed: $RES"
```
Comment (ashawley) [inline release.sh]: @non Yeah, the shell script overrides it here.

File: build.sbt (hunk 1 above)
Comment (ashawley) [reply build.sbt]: Yes, exactly.

**Comment by non** [comment]
👍

**Pull request merged**